### PR TITLE
Fix capitalized cols

### PR DESCRIPTION
**Pull request by utkarsharma2**
Handled uppercase col name issue with SnowflakeDB.

### CODE REVIEW

**Comment by dimberman** [comment]
@utkarsharma2 please add a fix to make the columns lowercase in sql_dataframe.py using `df.columns=df.colums.lower`

File: example_dags/example_snowflake_partial_table_with_append.py [hunk 1]
```diff
@@ -111,7 +111,13 @@ def example_snowflake_partial_table_with_append():
     append(
         append_table=filtered_data,
         columns=["sell", "list", "variable", "value"],
-        main_table=Table(table_name="homes_reporting"),
+        main_table=Table(
```
Comment (utkarsharma2) [inline example_dags/example_snowflake_partial_table_with_append.py:114]: @dimberman, I think we need to infer connection details from 1st table and pass it explicitly?

**Comment by tatiana** [comment]
@utkarsharma2 does this address issue: #154?

**Comment by utkarsharma2** [comment]
@tatiana - this is one of the solution, but the solution mentioned in ticket(#154) is generic one. @dimberman - can we merge this for the time being?

**Comment by tatiana** [comment]
@utkarsharma2 the generic implementation was merged (#215), how should we proceed with this PR?

**Pull request closed without merging**